Dockerfile

```diff
@@ -30,35 +33,54 @@ RUN apk --no-cache --update add \
     mariadb-client \
     ncurses \
     oniguruma-dev \
+    openssl \
     patch \
+    pcre2 \
+    perl \
     procps \
     py3-pip \
     py3-pygments \
-    python3-dev
+    python3-dev \
+    zlib
 
-# Build PHP extensions
+# Build PHP extensions and latest version of Git
 RUN apk --no-cache add --virtual build-dependencies \
         $PHPIZE_DEPS \
+        asciidoc \
+        build-base \
         curl-dev \
+        expat-dev \
         freetype-dev \
+        gettext \
         libjpeg-turbo-dev \
         libpng-dev \
         libzip-dev \
         mariadb-dev \
+        openssl-dev \
+        perl \
+        tar \
+        wget \
+        xmlto \
+        zlib-dev \
     && docker-php-ext-configure gd \
         --with-freetype \
         --with-jpeg \
     && docker-php-ext-install -j "$(nproc)" \
-        curl \
         gd \
-        iconv \
-        mbstring \
         mysqli \
         pcntl \
     && pecl install apcu-5.1.17 \
     && docker-php-ext-enable apcu \
     && pecl install zip-1.15.5 \
     && docker-php-ext-enable zip \
+    && wget https://mirrors.edge.kernel.org/pub/software/scm/git/git-${GIT_VERSION}.tar.gz \
```

Comment:
Could we put these install steps in a script so the Git install is a single line?